src/libraries/System.Management/src/System.Management.csproj

```diff
@@ -7,7 +7,7 @@
     <IsPackable>true</IsPackable>
     <!-- If you enable GeneratePackageOnBuild for this package and bump ServicingVersion, make sure to also bump ServicingVersion in Microsoft.Windows.Compatibility.csproj once for the next release. -->
```

Comment:
Both OOB packages that are getting built need Microsoft.Windows.Compatibility to be built as well.